### PR TITLE
Add Basic SSO Button Component With Configuration

### PR DESCRIPTION
**Pull request by mhoc**
Will need to get the `settings.dev.json` file from me in order for the app to work after this is merged. 

Login does not work completely because we need some user creation and signin hooks, but the sign-in portion does work.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 *.DS_Store
 .AppleDouble
 .LSOverride
+settings.*json
```
Comment (bwencke) [inline .gitignore:8]: Is there a reason you didn't include a settings file with empty fields?

**Review by bwencke — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (mhoc) [reply .gitignore:8]: Yup I can add that.

File: imports/api/Users/index.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ const UserSchema = new SimpleSchema({
   },
 })
 
-const Users = new Mongo.Collection('users')
+const Users = Meteor.users;
```
Comment (ashleyzeg) [inline imports/api/Users/index.js:42]: semi-colon???

**Review by ashleyzeg — APPROVED** [review]

File: imports/api/Users/index.js (hunk 2 above)
Comment (mhoc) [reply imports/api/Users/index.js:42]: I'm bi-polar.

**Pull request merged**